.goreleaser.yaml

```diff
@@ -42,127 +49,59 @@ snapshot:
-    # Default is empty.
-    replacements:
-      386: 32-bit
-      darwin: macOS
-
-    # Your app's vendor.
-    # Default is empty.
+  - package_name: upcloud-cli
+    file_name_template: "{{ .ConventionalFileName }}"
     vendor: UpCloudLtd.
-    # Your app's homepage.
-    # Default is empty.
     homepage: https://upcloud.com/
-
-    # Your app's maintainer (probably you).
-    # Default is empty.
     maintainer: UpCloud team <contact@upcloud.com>
-
-    # Your app's description.
-    # Default is empty.
     description: CLI tool for managing UpCloud services.
-
-    # Your app's license.
-    # Default is empty.
     license: MIT
-
-    # Formats to be generated.
     formats:
       - apk
       - deb
       - rpm
 
-    # Packages your package depends on.
-    # dependencies:
-    #   - bash-completion
-
     # Packages your package recommends installing.
     recommends:
       - bash-completion
 
-    overrides:
-      rpm:
-        replacements:
-          amd64: x86_64
-
-    # Custon configuration applied only to the RPM packager.
+    # Custom configuration applied only to the RPM packager.
     rpm:
       # The package group. This option is deprecated by most distros
       # but required by old distros like CentOS 5 / EL 5 and earlier.
       group: Unspecified
 
-      # # Compression algorithm.
-      # compression: lzma
-
-    # Custom configuration applied only to the Deb packager.
-    deb:
-      # Custom deb special files.
-
 brews:
-  -
-    # GOARM to specify which 32-bit arm version to use if there are multiple versions
+  - # GOARM to specify which 32-bit arm version to use if there are multiple versions
     # from the build section. Brew formulas support atm only one 32-bit version.
     # Default is 6 for all artifacts or each id if there a multiple versions.
     goarm: 6
 
     # NOTE: make sure the url_template, the token and given repo (github or gitlab) owner and name are from the
     # same kind. We will probably unify this in the next major version like it is done with scoop.
-
     # GitHub/GitLab repository to push the formula to
     # Gitea is not supported yet, but the support coming
-    tap:
+    repository:
       owner: UpCloudLtd
       name: homebrew-tap
+
       # Optionally a token can be provided, if it differs from the token provided to GoReleaser
       token: "{{ .Env.HOMEBREW_TAP_GITHUB_TOKEN }}"
-
-    # Allows you to set a custom download strategy. Note that you'll need
-    # to implement the strategy and add it to your tap repository.
-    # Example: https://docs.brew.sh/Formula-Cookbook#specifying-the-download-strategy-explicitly
-    # Default is empty.
-    # download_strategy: CurlDownloadStrategy.
-
-    # Git author used to commit to the repository.
-    # Defaults are shown.
     commit_author:
       name: goreleaserbot
       email: goreleaser@carlosbecker.com
 
-    # Folder inside the repository to put the formula.
-    # Default is the root folder.
-    folder: Formula
```

Comment:
Homebrew seems to [recommend](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap#creating-a-tap) using Formulas folder, but both should work.

Reply:
Using that instead, makes sense.